main.cc

```diff
@@ -1819,7 +1819,7 @@ sharded<locator::shared_token_metadata> token_metadata;
                 std::ref(feature_service), std::ref(mm), std::ref(token_metadata), std::ref(erm_factory),
                 std::ref(messaging), std::ref(repair),
                 std::ref(stream_manager), std::ref(lifecycle_notifier), std::ref(bm), std::ref(snitch),
-                std::ref(tablet_allocator), std::ref(cdc_generation_service), std::ref(view_builder), std::ref(qp), std::ref(sl_controller),
+                std::ref(tablet_allocator), std::ref(cdc_generation_service), std::ref(view_builder), std::ref(view_building_worker), std::ref(qp), std::ref(sl_controller),
```

Comment:
The list of things storage_service depends on is huge already. So (a late question, but still) -- why doesn't view_builder depend on view_builder_worker? It sounds (from services names) like the should depend on each other

Reply:
They are similar services but the view building worker (responsible for building tablet-based views) replaces view builder (responsible for building vnode-based views), so they shouldn't depend on each other.

Reply:
OK, thanks